### DISCUSSION TITLE
Alias visualization scale

### DISCUSSION THREAD

**Discussion by xPhilippeRoux** [HALRobotics/Beta#90] [Q&A] [answered]
I can't manage to find where to set the scale for aliases which are really small on my typical objects.

Can you help me ?

Thanks.

Philippe

**Comment by sebandraos** [comment, marked as answer]
Hi @XtreeE-PRO ,
The text size is tied to the Frame size in Grasshopper. Personally I find that a frame size between 50 and 100mm is pretty usable but obviously that's a personal preference.
![image](https://user-images.githubusercontent.com/6084391/112628353-b4525f80-8e2a-11eb-9d6c-6fff2506a47c.png)

**Reply by xPhilippeRoux** [reply]
Thanks !
